WordPress/Classes/Models/ReaderPost.m

```diff
@@ -178,5 +179,66 @@ - (NSURL *)featuredImageURLForDisplay
     return [self featuredImageURL];
 }
 
+- (SourceAttributionStyle)sourceAttributionStyle
+{
+    if (!self.sourceAttribution) {
+        return SourceAttributionStyleNone;
+    }
+    if ([self.sourceAttribution.attributionType isEqualToString:SourcePostAttributionTypePost]) {
+        return SourceAttributionStylePost;
+    } else if ([self.sourceAttribution.attributionType isEqualToString:SourcePostAttributionTypeSite]) {
+        return SourceAttributionStyleSite;
+    } else {
+        return SourceAttributionStyleNone;
+    }
```

Comment:
Nipticky comment: second *if* doesn't have else. Can we normalize and add else there (or remove the else from them all?).

^^ By all means, feel free to disregard please